corehq/motech/repeaters/models.py

```diff
@@ -372,7 +385,7 @@ def set_next_attempt(self):
             next_attempt_at=now + interval,
         )
 
-    def reset_next_attempt(self):
+    def reset_backoff(self):
         if self.last_attempt_at or self.next_attempt_at:
             self.last_attempt_at = None
```

Comment:
Backoff uses `last_attempt_at` to determine the duration of the next backoff interval.

Repeat records implement exponential backoff by multiplying the last interval by 3, and this code has preserved that behavior.

We set `last_attempt_at` to `None` so that the next time we need to back off, we know it is the initial interval and should be set it to the minimum. This matches [the way we use `RepeatRecord.last_checked` when setting `RepeatRecord.next_check`](https://github.com/dimagi/commcare-hq/blob/85bbfa337942042f5539d143b6b6238d44eb9a1f/corehq/motech/repeaters/models.py#L1106) for exponential backoff currently.

Reply:
Is that explained somewhere in a comment or docstring? If not maybe one could be added?

Reply:
b70fc523d63